src/gasman.h

```diff
@@ -378,9 +379,9 @@ extern void CHANGED_BAG_IMPL(Bag b);
 **  accidently use the same value for two different types.
 **
 **  <size> is the size of the new bag in bytes and must be a value  between 0
-**  and $2^{24}-1$.  The   application can find  the    size  of a   bag with
-**  'SIZE_BAG'    and  change   it  with  'ResizeBag'   (see   "SIZE_BAG" and
-**  "ResizeBag").
+**  and $2^{32}-1$ (on 32 bit systems)  resp. $2^{48}-1$ (on 64 bit systems).
```

Comment:
Sure there was a reason: In the original GASMAN version, the bag header on 32 bit systems had this layout:
```
typedef struct {
    uint8_t type : 8;
    uint32_t size : 24;
    Bag link;
    Bag data[];
} BagHeader;
```
So there were only 24 bits for the size field.

Since then (back in the 90ies), nobody adapted this comment.